app/controllers/api/users_controller.rb

```diff
@@ -3,7 +3,8 @@ class UsersController < Api::ApplicationController
     skip_before_action :doorkeeper_authorize!, only: %i[create]
 
     def create
-      user = User.new(name: user_params[:name], email: user_params[:email], password: user_params[:password])
+      user = User.new(name: user_params[:name], email: user_params[:email], password: user_params[:password],
+                      role: user_params[:role])
```

Comment:
- [ ] You are using Doorkeeper:

![image](https://user-images.githubusercontent.com/57421823/233588147-3971eb07-7e3e-4311-ab40-3e099c49d59a.png)

This means your app requires special procedures to be run:

![image](https://user-images.githubusercontent.com/57421823/233588311-19e318e0-01f1-4100-b4f1-2c5f3caa5be6.png)

Kindly, add specific instructions to both readmes explaining how to properly generate and set the `client_id` and the `client_secret` so both apps can communicate.

That way, your project will be well documented, and your users will be able to run both apps smoothly. 💪